styles/styles.css

```diff
@@ -0,0 +1,115 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans' rel="stylesheet");
+
+.container {
+  font-family: 'Noto Sans', sans-serif;
+  text-align: center;
+  width: 100%;
+  }
+
+ul {
+  list-style-type: none;
+}
+
+nav ul {
+  text-align: left;
+  font-weight: bold;
+  display: grid;
+  grid-template-columns: 60% 20% 20%;
+}
+
+.top {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.top a {
+  margin: 3em;
+  border: solid #9ea1a5 thin;
+  border-radius: 10px;
+  text-decoration: none;
+  color: black;
+  padding: 10px;
+  font-size: 1em;
+}
+
+.quote {
+  line-height: 2em;
+  font-family: serif;
+  font-style: italic;
+  font-size: 25px;
+  padding: 125px, 125px;
+}
+
+.quote p {
+  text-align: right;
+  padding-right: 10%;
+}
+
+.mission ul {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  text-align: left;
+}
+
+.mission img {
+  float: left;
+  margin-top: 0px;
+  padding-right: 5px;
+}
+
+.team ul {
+  width: 100%;
+  display: grid;
+  grid-template: auto auto / repeat(7, 1fr);
+}
+
+.team ul li span {
+  display: block;
+}
+.team ul li img {
+  border-radius: 50%;
+}
+
+.team ul li:first-of-type {
+  grid-column: 2;
+  grid-row: 1;
+}
+
+.team ul li:nth-of-type(2) {
+  grid-column: 4;
+  grid-row: 1;
+}
+
+.team ul li:nth-of-type(3) {
+  grid-column: 6;
+  grid-row: 1;
+}
+
+.team ul li:nth-of-type(4) {
+  grid-column: 3;
+  grid-row: 2;
+}
+
+.team ul li:nth-of-type(5) {
+  grid-column: 5;
+  grid-row: 2;
+}
+
+
+.sponsor ul {
```

Comment:
I like how you used the 'space-around' value to keep your sponsor boxes tidy, regardless of the screen size.  Excellent job!